overlord/devicestate/firstboot.go

```diff
@@ -375,6 +384,34 @@ func importAssertionsFromSeed(st *state.State, sysLabel string, isCoreBoot bool)
 	return deviceSeed, nil
 }
 
+// processAutoImportAssertions attempts to load the auto import assertions 
+// and create all knows system users, if and only if the model grade is dangerous.
+// Processing of the auto-import assertion is opportunistic and should not fail
+// if model grade is dangerous, it attempts to load
+// auto import assertions and create all knows system users.
+// Processing of the auto-import assertion is opportunistic and should not fail
```

Comment:
this seems to repeat some things a couple of times?

Reply:
ups, GitHub merging Philip's suggestion kept old text, removed now.